apps/builder/app/builder/features/style-panel/shared/css-value-input/parse-intermediate-or-invalid-value.ts

```diff
@@ -23,22 +23,32 @@ export const parseIntermediateOrInvalidValue = (
     };
   }
 
-  // Try value with existing or fallback unit
-  const unit = "unit" in styleValue ? styleValue.unit ?? "px" : "px";
-  let styleInput = parseCssValue(property, `${value}${unit}`);
+  const valueInfo = properties[property as keyof typeof properties];
 
-  if (styleInput.type !== "invalid") {
-    return styleInput;
+  // - When user enters a number, we don't know if its a valid unit value,
```

Comment:
can we leverage unitGroups for this? Seem equivalent to types.

Reply:
they are different, they just have some overlap

Reply:
Isn't the point of this change to detect unit?

Reply:
No the point of this pr is to stop guessing the unit when value can be anything, because with custom identifiers we can't guess it, anything is valid